vault-github/run.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
This improves portability, but there's some bashisms we should get rid off. PS. Does this repo already use shftm/shellcheck? If not, might be a good addition.

```suggestion
#!/usr/bin/env sh
```